integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/freeipa/FreeIpaUpgradeTests.java

```diff
@@ -219,4 +254,16 @@ private boolean isSuccess(OperationStatus operationStatus) {
         OperationState operationState = operationStatus.getStatus();
         return COMPLETED == operationState;
     }
+
+    private void kinit(SdxTestDto sdxTestDto, SdxClient sdxClient) {
+        sshJClientActions.checkKinitDuringFreeipaUpgrade(sdxTestDto, getInstanceGroups(sdxTestDto, sdxClient),
+                List.of(HostGroupType.MASTER.getName()));
```

Comment:
please add try-catch for every exception and add a message to be more easy to find the issue, also please log it